apps/vr-tests/screener.local.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   projectRepo: 'microsoft/fluentui',
   storybookConfigDir: '.storybook',
-  apiKey: 'ff2096ee-7c7d-4e80-824b-9114ff43549c',
+  apiKey: '[Screener Key here]',
```

Comment:
Maybe do this so you can pass it in on the command line? 
```suggestion
  apiKey: process.env.SCREENER_API_KEY,
```